pages/api/mentors/index.js

```diff
@@ -10,16 +10,30 @@ export default async function handler(req, res) {
     return;
   }
 
+  // this does not work:
+  // $includesAll: [{ $contains: "mentor" }, { $contains: "test" }],
```

Comment:
Ah, that's a good one. Yes, that means that each value is both "mentor" and "test", which cannot be.

I think you found the right solution already, essentially:
```
$all: [{
   roles: { $contains: "mentor" },
}, {
   roles: { $contains: "test"}
}]
```

Reply:
it was confusing because `$contains` should apply to the array, it's different from `$is` and both could be true. Anyway, after playing around a bit I found the solution ^^ thanks for validating it!